packages/gatsby-plugin-static-image/src/babel-parse-to-ast.ts

```diff
@@ -0,0 +1,64 @@
+import { parse, ParserOptions } from "@babel/parser"
+import babel from "@babel/core"
+
+const PARSER_OPTIONS: ParserOptions = {
+  allowImportExportEverywhere: true,
+  allowReturnOutsideFunction: true,
+  allowSuperOutsideMethod: true,
+  sourceType: `unambiguous`,
+  plugins: [
+    `jsx`,
+    `flow`,
+    `doExpressions`,
+    `objectRestSpread`,
+    [
+      `decorators`,
+      {
+        decoratorsBeforeExport: true,
+      },
+    ],
+    `classProperties`,
+    `classPrivateProperties`,
+    `classPrivateMethods`,
+    `exportDefaultFrom`,
+    `exportNamespaceFrom`,
+    `asyncGenerators`,
+    `functionBind`,
+    `functionSent`,
+    `dynamicImport`,
+    `numericSeparator`,
+    `optionalChaining`,
+    `importMeta`,
+    `bigInt`,
+    `optionalCatchBinding`,
+    `throwExpressions`,
+    [
+      `pipelineOperator`,
+      {
+        proposal: `minimal`,
+      },
+    ],
+    `nullishCoalescingOperator`,
+  ],
+}
+
+export function getBabelParserOptions(filePath: string): ParserOptions {
+  // Flow and TypeScript plugins can't be enabled simultaneously
```

Comment:
❓ is there any place I can check for why?

Reply:
They have conflicting behaviour. That said, this is just lifted from the other plugin, so it may not be needed.